diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 089384b9f83b648792aaa49e0c19bfe2e84f5817..3b2ad609e1519f2d8ad9fbe036b1039bba189fa4 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -283,6 +283,7 @@ namespace enterprise_connectors {
 class LinuxKeyRotationCommand;
 }  // namespace enterprise_connectors
 namespace extensions {
+class ComponentLoader;
 class InstalledLoader;
 class UnpackedInstaller;
 }  // namespace extensions
@@ -618,6 +619,7 @@ class BASE_EXPORT ScopedAllowBlocking {
   friend class cronet::CronetPrefsManager;
   friend class crypto::ScopedAllowBlockingForNSS;  // http://crbug.com/59847
   friend class drive::FakeDriveService;
+  friend class extensions::ComponentLoader;
   friend class extensions::InstalledLoader;
   friend class extensions::UnpackedInstaller;
   friend class font_service::internal::MappedFontFile;
