diff --git a/chrome/browser/ocbot/BUILD.gn b/chrome/browser/ocbot/BUILD.gn
new file mode 100644
index 0000000000000000000000000000000000000000..bbfaeb6c86d579bf49e57628eef3e2c991b7b7d6
--- /dev/null
+++ b/chrome/browser/ocbot/BUILD.gn
@@ -0,0 +1,30 @@
+# Copyright 2025 The Ocbot Authors
+# SPDX-License-Identifier: MIT
+
+source_set("ocbot") {
+  sources = [
+    "browser_updater.h",
+    "extension_updater.cc",
+    "extension_updater.h",
+  ]
+
+  if (is_mac) {
+    sources += [ "browser_updater.mm" ]
+  } else {
+    sources += [ "browser_updater.cc" ]
+  }
+
+  deps = [
+    "//base",
+    "//chrome/browser/profiles:profile",
+    "//chrome/common",
+    "//components/version_info",
+    "//content/public/browser",
+    "//extensions/browser",
+    "//extensions/common",
+    "//net",
+    "//services/network/public/cpp",
+    "//services/network/public/mojom",
+    "//third_party/zlib/google:zip",
+  ]
+}
