diff --git a/chrome/browser/ocbot/BUILD.gn b/chrome/browser/ocbot/BUILD.gn
new file mode 100644
index 0000000000000000000000000000000000000000..514c66c4c796464e3702d95cdf5e1e905ab3783a
--- /dev/null
+++ b/chrome/browser/ocbot/BUILD.gn
@@ -0,0 +1,45 @@
+# Copyright 2025 The Ocbot Authors
+# SPDX-License-Identifier: MIT
+
+source_set("ocbot") {
+  sources = [
+    "browser_updater.h",
+    "extension_updater.cc",
+    "extension_updater.h",
+    "runtime_manager.cc",
+    "runtime_manager.h",
+    "runtime_updater.cc",
+    "runtime_updater.h",
+    "skill_installer.cc",
+    "skill_installer.h",
+  ]
+
+  if (is_mac) {
+    sources += [
+      "browser_updater.mm",
+      "ocbot_status_icon_mac.h",
+      "ocbot_status_icon_mac.mm",
+    ]
+  } else {
+    sources += [ "browser_updater.cc" ]
+  }
+
+  deps = [
+    "//base",
+    "//chrome/common",
+    "//components/keep_alive_registry",
+    "//components/version_info",
+    "//content/public/browser",
+    "//crypto",
+    "//extensions/browser",
+    "//extensions/common",
+    "//net",
+    "//services/network/public/cpp",
+    "//services/network/public/mojom",
+    "//third_party/zlib/google:zip",
+  ]
+
+  if (is_mac) {
+    frameworks = [ "UserNotifications.framework" ]
+  }
+}
