diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 78b7dfb77ab164f035ede4152ebc2a818a30df94..3b3f37f5480ea52480f81a8541153e1eae3a0acd 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -110,6 +110,7 @@
 #include "ui/base/interaction/element_identifier.h"
 #include "ui/base/l10n/l10n_util.h"
 #include "ui/base/metadata/metadata_impl_macros.h"
+#include "ui/base/resource/resource_bundle.h"
 #include "ui/base/theme_provider.h"
 #include "ui/base/ui_base_features.h"
 #include "ui/base/window_open_disposition.h"
@@ -409,14 +410,6 @@ void ToolbarView::Init() {
   pinned_toolbar_actions_container_ = container_view_->AddChildView(
       std::make_unique<PinnedToolbarActionsContainer>(browser_view_, this));
 
-  if (features::HasTabSearchToolbarButton()) {
-    tab_search_button_ =
-        pinned_toolbar_actions_container()->CreatePermanentButtonFor(
-            kActionTabSearch);
-    tab_search_button_->SetProperty(views::kElementIdentifierKey,
-                                    kTabSearchButtonElementId);
-  }
-
   if (IsChromeLabsEnabled()) {
     UpdateChromeLabsNewBadgePrefs(browser_->profile());
 
