diff --git a/chrome/browser/extensions/api/BUILD.gn b/chrome/browser/extensions/api/BUILD.gn
index 2c8ce77066767c1e0ba926429b69ed1d38c23a69..6063ddcec265cdd39c5c47bfbf552984a3db808a 100644
--- a/chrome/browser/extensions/api/BUILD.gn
+++ b/chrome/browser/extensions/api/BUILD.gn
@@ -68,6 +68,7 @@ group("api_implementations") {
       "//chrome/browser/extensions/api/command_line_private",
       "//chrome/browser/extensions/api/content_settings",
       "//chrome/browser/extensions/api/experimental_actor",
+      "//chrome/browser/extensions/api/ocbot",
       "//chrome/browser/extensions/api/experimental_ai_data",
       "//chrome/browser/extensions/api/idltest",
       "//chrome/browser/extensions/api/networking_private",
