diff --git a/chrome/app/theme/chrome_unscaled_resources.grd b/chrome/app/theme/chrome_unscaled_resources.grd
index 590a4a77ba9222b1f350967af801897de30b95a6..703822a8f2f168f8f5c111ea4dc30bf0d64313f1 100644
--- a/chrome/app/theme/chrome_unscaled_resources.grd
+++ b/chrome/app/theme/chrome_unscaled_resources.grd
@@ -93,6 +93,7 @@
             <include name="IDR_PRODUCT_LOGO_24_SHORTCUTS" file="chromium/product_logo_24.png" type="BINDATA" />
             <include name="IDR_PRODUCT_LOGO_64_SHORTCUTS" file="chromium/product_logo_64.png" type="BINDATA" />
             <include name="IDR_PRODUCT_LOGO_128_SHORTCUTS" file="chromium/product_logo_128.png" type="BINDATA" />
+            <include name="IDR_OCBOT_TOOLBAR_ICON" file="chromium/ocbot_toolbar_icon.png" type="BINDATA" />
         </else>
       </if> <!-- _google_chrome -->
       <!-- NOTE: product_logo_*.* files beyond what's listed above are
