diff --git a/chrome/app/theme/chrome_unscaled_resources.grd b/chrome/app/theme/chrome_unscaled_resources.grd
index 18fdde153e08a72eb3d60ba91a8eee023e58253c..d0de4f52987f0580c3b94d97d385f97bf0e258f2 100644
--- a/chrome/app/theme/chrome_unscaled_resources.grd
+++ b/chrome/app/theme/chrome_unscaled_resources.grd
@@ -91,6 +91,7 @@
             <include name="IDR_PRODUCT_LOGO_24_SHORTCUTS" file="chromium/product_logo_24.png" type="BINDATA" />
             <include name="IDR_PRODUCT_LOGO_64_SHORTCUTS" file="chromium/product_logo_64.png" type="BINDATA" />
             <include name="IDR_PRODUCT_LOGO_128_SHORTCUTS" file="chromium/product_logo_128.png" type="BINDATA" />
+            <include name="IDR_OCBOT_TOOLBAR_ICON" file="chromium/ocbot_toolbar_icon.png" type="BINDATA" />
         </else>
       </if> <!-- _google_chrome -->
       <!-- NOTE: product_logo_*.* files beyond what's listed above are
