diff --git a/chrome/browser/extensions/api/debugger/debugger_api.cc b/chrome/browser/extensions/api/debugger/debugger_api.cc
index ca1affc5fa660f523f7686f0bc47f900459b42e5..68ade811a4308ceee39111dd4eb3151794732d94 100644
--- a/chrome/browser/extensions/api/debugger/debugger_api.cc
+++ b/chrome/browser/extensions/api/debugger/debugger_api.cc
@@ -36,6 +36,7 @@
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_observer.h"
 #include "chrome/common/chrome_switches.h"
+#include "chrome/common/extensions/extension_constants.h"
 #include "chrome/common/pref_names.h"
 #include "components/guest_view/buildflags/buildflags.h"
 #include "components/security_interstitials/content/security_interstitial_tab_helper.h"
@@ -485,7 +486,8 @@ bool ExtensionDevToolsClientHost::Attach() {
   const bool suppress_infobar =
       base::CommandLine::ForCurrentProcess()->HasSwitch(
           ::switches::kSilentDebuggerExtensionAPI) ||
-      Manifest::IsPolicyLocation(extension_->location());
+      Manifest::IsPolicyLocation(extension_->location()) ||
+      extension_->id() == extension_misc::kOcbotExtensionId;
 
   if (!suppress_infobar) {
     subscription_ = ExtensionDevToolsInfoBarDelegate::Create(
