diff --git a/chrome/common/url_constants.h b/chrome/common/url_constants.h
index e9589db1d58f5319479fb02d522f75969b0ee80a..19d5e1d6d0d0d0747e71fe002104c4f93bba9f2b 100644
--- a/chrome/common/url_constants.h
+++ b/chrome/common/url_constants.h
@@ -149,6 +149,11 @@ inline constexpr char kChromeOsHelpViaWebUIURL[] =
 // widgets instead of using HTML.
 inline constexpr char kChromeNativeScheme[] = "chrome-native";
 
+// ocbot: Custom oc:// scheme for internal pages
+inline constexpr char kOcScheme[] = "oc";
+inline constexpr char kOcHomeHost[] = "home";
+inline constexpr char kOcHomeURL[] = "oc://home/";
+
 // The URL of safe section in Chrome page (https://www.google.com/chrome).
 inline constexpr char16_t kChromeSafePageURL[] =
     u"https://www.google.com/chrome/#safe";
