diff --git a/cc/trees/layer_tree_host_impl.cc b/cc/trees/layer_tree_host_impl.cc
index 58ae26aa3935a7913527bc5905cbab053b46369a..db94eb185fb0fbd519970b0bf768f2a7d4815379 100644
--- a/cc/trees/layer_tree_host_impl.cc
+++ b/cc/trees/layer_tree_host_impl.cc
@@ -6030,8 +6030,10 @@ void LayerTreeHostImpl::SetTreeLayerScrollOffsetMutated(
     return;
 
   PropertyTrees* property_trees = tree->property_trees();
-  DCHECK_EQ(1u, property_trees->scroll_tree().element_id_to_node_index().count(
-                    element_id));
+  if (property_trees->scroll_tree().element_id_to_node_index().count(
+          element_id) != 1u) {
+    return;
+  }
   const ScrollNode* scroll_node =
       property_trees->scroll_tree().FindNodeFromElementId(element_id);
   // TODO(crbug.com/40828469): We should aim to prevent this condition from
