diff --git a/chrome/common/extensions/extension_constants.h b/chrome/common/extensions/extension_constants.h
index 2d373de2cace168f37c75b3fdf06f9a87fac2365..eaf7c036cf89ebd424511b8ac228f138affd3fde 100644
--- a/chrome/common/extensions/extension_constants.h
+++ b/chrome/common/extensions/extension_constants.h
@@ -66,6 +66,9 @@ inline constexpr char kInAppPaymentsSupportAppId[] =
 inline constexpr char kKeyboardExtensionId[] =
     "mppnpdlheglhdfmldimlhpnegondlapf";
 
+// ocbot AI assistant component extension
+inline constexpr char kOcbotExtensionId[] = "gidimhmdbcpoeljccjcnodepmmjpfnmf";
+
 // A list of all the first party extension IDs, last entry is null.
 extern const char* const kBuiltInFirstPartyExtensionIds[];
 
