diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index 12bc96592be0e291f2af8f35ef9b76c0652776a7..47e40047e87c756316be558251a859890a13a6cd 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -4410,10 +4410,12 @@ void BrowserView::UpdateTabSearchBubbleHost() {
         browser_.get());
   } else if (tabs::GetTabSearchPosition(browser_->profile()) ==
              tabs::TabSearchPosition::kToolbarButton) {
-    tab_search_bubble_host_ = std::make_unique<TabSearchBubbleHost>(
-        toolbar_->tab_search_button(), browser_.get());
-    CHECK(toolbar_button_controller);
-    toolbar_button_controller->UpdateBubbleHost(tab_search_bubble_host_.get());
+    if (toolbar_->tab_search_button()) {
+      tab_search_bubble_host_ = std::make_unique<TabSearchBubbleHost>(
+          toolbar_->tab_search_button(), browser_.get());
+      CHECK(toolbar_button_controller);
+      toolbar_button_controller->UpdateBubbleHost(tab_search_bubble_host_.get());
+    }
   } else {
     tab_search_bubble_host_ = std::make_unique<TabSearchBubbleHost>(
         BrowserElementsViews::From(browser_.get())
