diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 1f0f86c655ec8d13fec7c85eb7295419d93c3e9a..f7fa0e0edcf80f8a66ba22e1d9bb57fbdeddcf16 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -517,6 +517,7 @@ source_set("extensions") {
     "//chrome/browser/download",
     "//chrome/browser/extensions:cws_item_service_proto",
     "//chrome/browser/extensions:external_loader",
+    "//chrome/browser/ocbot",
     "//chrome/browser/extensions/api:api_registration",
     "//chrome/browser/favicon",
     "//chrome/browser/google",
