diff --git a/chrome/browser/ui/startup/infobar_utils.cc b/chrome/browser/ui/startup/infobar_utils.cc
index e02a21847591a98d637fe129687e504d487dda9d..fd23b94c3b0ab40bebb8eb7953b76172b9fffaf5 100644
--- a/chrome/browser/ui/startup/infobar_utils.cc
+++ b/chrome/browser/ui/startup/infobar_utils.cc
@@ -176,9 +176,10 @@ void AddInfoBarsIfNecessary(BrowserWindowInterface* browser,
   infobars::ContentInfoBarManager* infobar_manager =
       infobars::ContentInfoBarManager::FromWebContents(web_contents);
 
-  if (!google_apis::HasAPIKeyConfigured()) {
-    GoogleApiKeysInfoBarDelegate::Create(infobar_manager);
-  }
+  // OcBot: suppress "Google API keys are missing" infobar
+  // if (!google_apis::HasAPIKeyConfigured()) {
+  //   GoogleApiKeysInfoBarDelegate::Create(infobar_manager);
+  // }
 
   if (ObsoleteSystem::IsObsoleteNowOrSoon()) {
     PrefService* local_state = g_browser_process->local_state();
