diff --git a/chrome/browser/ui/startup/startup_browser_creator_impl.cc b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
index eee0ccf1b6dd897158b7ebb993bc3863d21f83c6..8819d5882354a90f7af2ad75602a832ed409270c 100644
--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -57,6 +57,7 @@
 #include "chrome/common/chrome_switches.h"
 #include "chrome/common/chrome_version.h"
 #include "chrome/common/pref_names.h"
+#include "chrome/common/url_constants.h"
 #include "chrome/common/webui_url_constants.h"
 #include "components/custom_handlers/protocol_handler_registry.h"
 #include "components/prefs/pref_service.h"
@@ -501,7 +502,8 @@ StartupBrowserCreatorImpl::DetermineStartupTabs(
       return {std::move(tabs), launch_result};
     }
 
-    return {StartupTabs({StartupTab(GURL(chrome::kChromeUINewTabURL))}),
+    // ocbot: Use oc://home instead of chrome://newtab
+    return {StartupTabs({StartupTab(GURL(chrome::kOcHomeURL))}),
             launch_result};
   }
 
@@ -637,7 +639,7 @@ Browser* StartupBrowserCreatorImpl::RestoreOrCreateBrowser(
   browser = OpenTabsInBrowser(
       browser, process_startup,
       (tabs.empty()
-           ? StartupTabs({StartupTab(GURL(chrome::kChromeUINewTabURL))})
+           ? StartupTabs({StartupTab(GURL(chrome::kOcHomeURL))})
            : tabs),
       (behavior == BrowserOpenBehavior::USE_EXISTING_AND_OVERWRITE_ACTIVE_TAB
            ? (TabOverWrite::kYes)
@@ -679,7 +681,8 @@ StartupBrowserCreatorImpl::DetermineBrowserOpenBehavior(
     // Don't perform a session restore on a post-crash launch, as this could
     // cause a crash loop.
     if (!(options & IS_POST_CRASH_LAUNCH)) {
-      return BrowserOpenBehavior::SYNCHRONOUS_RESTORE;
+      // ocbot: Don't session-restore, always start with oc://home
+      return BrowserOpenBehavior::NEW;
     }
   }
 
