diff --git a/third_party/blink/renderer/platform/BUILD.gn b/third_party/blink/renderer/platform/BUILD.gn
index 49d07b1c0cbebcdb4d7587d1d564ba7c73939007..4af31deabc745bde6f0efda27201b89c7914bccf 100644
--- a/third_party/blink/renderer/platform/BUILD.gn
+++ b/third_party/blink/renderer/platform/BUILD.gn
@@ -175,6 +175,7 @@ group("make_platform_generated") {
     "//third_party/blink/public/common:buildflags",
     "//third_party/blink/renderer/platform/heap:blink_heap_buildflags",
     "//third_party/blink/renderer/platform/network:make_generated",
+    "//oc_fingerprint",
   ]
 }
 
@@ -1855,6 +1856,7 @@ component("platform") {
     "//ui/gfx/mojom",
     "//ui/gl:buildflags",
     "//ui/native_theme",
+    "//oc_fingerprint",
   ]
 
   if (has_zstd_compression) {
