diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index 924200dec9703311a7c6ee21d5d0c501a0738c17..e3fb8e84c830393a5ecf890d669c1dbe7fdcea82 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -55,6 +55,7 @@ enum VisitSource {
   SOURCE_SAFARI_IMPORTED = 5,
   SOURCE_ACTOR = 6,
   SOURCE_OS_MIGRATION_IMPORTED = 7,
+  SOURCE_CHROME_IMPORTED = 8,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
