diff --git a/chrome/browser/extensions/api/ocbot/ocbot_api.h b/chrome/browser/extensions/api/ocbot/ocbot_api.h
new file mode 100644
index 0000000000000000000000000000000000000000..323781014c972c03e1cfd497054f6ce12c5224ba
--- /dev/null
+++ b/chrome/browser/extensions/api/ocbot/ocbot_api.h
@@ -0,0 +1,36 @@
+// Copyright 2025 The Ocbot Authors
+// SPDX-License-Identifier: MIT
+
+#ifndef CHROME_BROWSER_EXTENSIONS_API_OCBOT_OCBOT_API_H_
+#define CHROME_BROWSER_EXTENSIONS_API_OCBOT_OCBOT_API_H_
+
+#include "base/values.h"
+#include "extensions/browser/extension_function.h"
+
+namespace extensions {
+
+class OcbotSetProviderFunction : public ExtensionFunction {
+ public:
+  DECLARE_EXTENSION_FUNCTION("ocbot.setProvider", OCBOT_SETPROVIDER)
+  OcbotSetProviderFunction() = default;
+
+ private:
+  ~OcbotSetProviderFunction() override = default;
+  ResponseAction Run() override;
+  void OnSetProviderDone(bool success);
+};
+
+class OcbotGetConfigFunction : public ExtensionFunction {
+ public:
+  DECLARE_EXTENSION_FUNCTION("ocbot.getConfig", OCBOT_GETCONFIG)
+  OcbotGetConfigFunction() = default;
+
+ private:
+  ~OcbotGetConfigFunction() override = default;
+  ResponseAction Run() override;
+  void OnGetConfigDone(base::Value::Dict result);
+};
+
+}  // namespace extensions
+
+#endif  // CHROME_BROWSER_EXTENSIONS_API_OCBOT_OCBOT_API_H_
