diff --git a/components/os_crypt/common/keychain_password_mac.mm b/components/os_crypt/common/keychain_password_mac.mm
index caa0e420956a31be1bd744319a0e40a0e50b46cf..eab896ef2fa02460ae3600ffb018ad4cb3ae9a20 100644
--- a/components/os_crypt/common/keychain_password_mac.mm
+++ b/components/os_crypt/common/keychain_password_mac.mm
@@ -35,8 +35,8 @@
 const char kDefaultServiceName[] = "Chrome Safe Storage";
 const char kDefaultAccountName[] = "Chrome";
 #else
-const char kDefaultServiceName[] = "Chromium Safe Storage";
-const char kDefaultAccountName[] = "Chromium";
+const char kDefaultServiceName[] = "Ocbot Safe Storage";
+const char kDefaultAccountName[] = "Ocbot";
 #endif
 
 // These values are persisted to logs. Entries should not be renumbered and
