diff --git a/chrome/app/vector_icons/BUILD.gn b/chrome/app/vector_icons/BUILD.gn
index 7537afaf13946576de5a7cd9083aeec0d2434ad3..666831601908ed6bb1bf7c6387dc38b16999ffbf 100644
--- a/chrome/app/vector_icons/BUILD.gn
+++ b/chrome/app/vector_icons/BUILD.gn
@@ -139,6 +139,7 @@ aggregate_vector_icons("chrome_vector_icons") {
     "new_tab_refresh.icon",
     "new_window.icon",
     "notes.icon",
+    "ocbot_pointer_hand.icon",
     "open_in_browser.icon",
     "open_in_full.icon",
     "open_in_new.icon",
