diff --git a/chrome/browser/ocbot/BUILD.gn b/chrome/browser/ocbot/BUILD.gn
new file mode 100644
index 0000000000000000000000000000000000000000..b2ae79c1e0c045da6e1c5a2390f1e029033da88c
--- /dev/null
+++ b/chrome/browser/ocbot/BUILD.gn
@@ -0,0 +1,25 @@
+# Copyright 2025 The Ocbot Authors
+# SPDX-License-Identifier: MIT
+
+source_set("ocbot") {
+  sources = [
+    "browser_updater.h",
+    "browser_updater.mm",
+    "extension_updater.cc",
+    "extension_updater.h",
+  ]
+
+  deps = [
+    "//base",
+    "//chrome/browser/profiles:profile",
+    "//chrome/common",
+    "//components/version_info",
+    "//content/public/browser",
+    "//extensions/browser",
+    "//extensions/common",
+    "//net",
+    "//services/network/public/cpp",
+    "//services/network/public/mojom",
+    "//third_party/zlib/google:zip",
+  ]
+}
