diff --git a/third_party/blink/renderer/core/frame/navigator_device_memory.cc b/third_party/blink/renderer/core/frame/navigator_device_memory.cc
index 05fa33e4568e8eea926a0d97c1d83a4000fa6b3f..23d6a84d4c68c9925406229d1f4a43f15f7340f7 100644
--- a/third_party/blink/renderer/core/frame/navigator_device_memory.cc
+++ b/third_party/blink/renderer/core/frame/navigator_device_memory.cc
@@ -8,10 +8,15 @@
 #include "third_party/blink/public/mojom/use_counter/metrics/web_feature.mojom-shared.h"
 #include "third_party/blink/renderer/core/dom/document.h"
 #include "third_party/blink/renderer/core/frame/local_dom_window.h"
+#include "oc_fingerprint/fingerprint_manager.h"
 
 namespace blink {
 
 float NavigatorDeviceMemory::deviceMemory() const {
+  auto* fp = oc_fingerprint::FingerprintManager::GetInstance();
+  if (fp && fp->GetConfig().device_memory) {
+    return *fp->GetConfig().device_memory;
+  }
   return ApproximatedDeviceMemory::GetApproximatedDeviceMemory();
 }
 
