diff --git a/components/embedder_support/BUILD.gn b/components/embedder_support/BUILD.gn
index 660aeb47d9b125adad423e27c68541044c61191a..cbd8ccd41ec25fc5b0553d34e7fe0ce4a122646c 100644
--- a/components/embedder_support/BUILD.gn
+++ b/components/embedder_support/BUILD.gn
@@ -65,6 +65,7 @@ static_library("user_agent") {
     "//net",
     "//third_party/blink/public/common:headers",
     "//ui/base",
+    "//oc_fingerprint",
   ]
 }
 
