diff --git a/components/embedder_support/user_agent_utils.cc b/components/embedder_support/user_agent_utils.cc
index 857d7806537545ba98aec06bc87340f29c3f512d..55053606c2ca3228f63213961c40eca49e241758 100644
--- a/components/embedder_support/user_agent_utils.cc
+++ b/components/embedder_support/user_agent_utils.cc
@@ -474,11 +474,19 @@ std::string BuildOSCpuInfo(
 
 std::string GetProductAndVersion(
     UserAgentReductionEnterprisePolicyState user_agent_reduction) {
-  return ShouldReduceUserAgentMinorVersion(user_agent_reduction)
-             ? version_info::GetProductNameAndVersionForReducedUserAgent(
-                   blink::features::kUserAgentFrozenBuildVersion.Get())
-             : std::string(
-                   version_info::GetProductNameAndVersionForUserAgent());
+  std::string product_and_version =
+      ShouldReduceUserAgentMinorVersion(user_agent_reduction)
+          ? version_info::GetProductNameAndVersionForReducedUserAgent(
+                blink::features::kUserAgentFrozenBuildVersion.Get())
+          : std::string(
+                version_info::GetProductNameAndVersionForUserAgent());
+
+  // Force the return value to start with "Chrome/" instead of the actual product name (OcBot).
+  size_t slash_pos = product_and_version.find('/');
+  if (slash_pos != std::string::npos) {
+    return "Chrome" + product_and_version.substr(slash_pos);
+  }
+  return product_and_version;
 }
 
 std::optional<std::string> GetUserAgentFromCommandLine() {
