diff --git a/third_party/blink/public/strings/permission_element_strings.grd b/third_party/blink/public/strings/permission_element_strings.grd
index 8a937fe45b164af0b026da07d547f8f8cb0a5c27..fac828b3707d890d64798e731c372b7cc7decaee 100644
--- a/third_party/blink/public/strings/permission_element_strings.grd
+++ b/third_party/blink/public/strings/permission_element_strings.grd
@@ -230,6 +230,9 @@
       <message name="IDS_PERMISSION_REQUEST_INSTALL_WITH_HOST" desc="Label for a permission element that will trigger an installation request for an application whose host is specified">
           Install <ph name="HOSTNAME">$1<ex>www.example.com</ex></ph>
       </message>
+      <message name="IDS_PERMISSION_REQUEST_LAUNCH" desc="Label for a permission element that will trigger a launch request">
+          Launch
+      </message>
     </messages>
   </release>
 </grit>
