diff --git a/services/network/public/mojom/network_context.mojom b/services/network/public/mojom/network_context.mojom
index 709aae5568b73df28282dec1d0a87ca4a0cb054c..1ac43c4f30638db293b6814a5fbfd4413d43c4f4 100644
--- a/services/network/public/mojom/network_context.mojom
+++ b/services/network/public/mojom/network_context.mojom
@@ -591,6 +591,10 @@ struct NetworkContextParams {
   // gets applied while constructing NetworkContext by updating the quic_params.
   int64? quic_idle_connection_timeout_seconds;
 
+  // HTTP/2 SETTINGS parameters to override.
+  // Keys are SpdySettingsId (uint32), values are the setting values (uint32).
+  map<uint32, uint32>? http2_settings;
+
   // If true, the HTTP cache will be encrypted on disk. For now, this only
   // switches the backend on Windows.
   bool enable_encrypted_http_cache = false;
