diff --git a/chrome/common/extensions/api/api_sources.gni b/chrome/common/extensions/api/api_sources.gni
index 9a041705164b257ed12bf70797b905506b5a28d7..4798ffd1dae5297b7378bc8b43c3c12fbb2f0f3a 100644
--- a/chrome/common/extensions/api/api_sources.gni
+++ b/chrome/common/extensions/api/api_sources.gni
@@ -69,6 +69,7 @@ if (enable_extensions) {
     "accessibility_features.json",
     "experimental_actor.idl",
     "experimental_ai_data.idl",
+    "ocbot.idl",
     "autofill_private.idl",
     "autotest_private.idl",
     "bookmark_manager_private.json",
