diff --git a/oc_fingerprint/BUILD.gn b/oc_fingerprint/BUILD.gn
new file mode 100644
index 0000000000000000000000000000000000000000..923c88d2f5264dd69f38e2daf6402116231b381d
--- /dev/null
+++ b/oc_fingerprint/BUILD.gn
@@ -0,0 +1,18 @@
+# Copyright 2025 The Chromium Authors
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+component("oc_fingerprint") {
+  sources = [
+    "config_parser.cc",
+    "config_parser.h",
+    "fingerprint_manager.cc",
+    "fingerprint_manager.h",
+  ]
+
+  defines = [ "OC_FINGERPRINT_IMPLEMENTATION" ]
+
+  deps = [
+    "//base",
+  ]
+}
