diff --git a/chrome/browser/ui/views/frame/browser_view.cc b/chrome/browser/ui/views/frame/browser_view.cc
index d0cac58da0356babfb5adc80032675d790e779ec..221af145c19b0c2672ac3b29b3a102567dd63968 100644
--- a/chrome/browser/ui/views/frame/browser_view.cc
+++ b/chrome/browser/ui/views/frame/browser_view.cc
@@ -4594,10 +4594,12 @@ void BrowserView::UpdateTabSearchBubbleHost() {
         vertical_tab_strip_container_->GetTopContainer()->GetTabSearchButton(),
         browser_.get());
   } else if (features::HasTabSearchToolbarButton()) {
-    tab_search_bubble_host_ = std::make_unique<TabSearchBubbleHost>(
-        toolbar_->tab_search_button(), browser_.get());
-    CHECK(toolbar_button_controller);
-    toolbar_button_controller->UpdateBubbleHost(tab_search_bubble_host_.get());
+    if (toolbar_->tab_search_button()) {
+      tab_search_bubble_host_ = std::make_unique<TabSearchBubbleHost>(
+          toolbar_->tab_search_button(), browser_.get());
+      CHECK(toolbar_button_controller);
+      toolbar_button_controller->UpdateBubbleHost(tab_search_bubble_host_.get());
+    }
   } else {
     tab_search_bubble_host_ = std::make_unique<TabSearchBubbleHost>(
         BrowserElementsViews::From(browser_.get())
