diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 35746fea54a485ae74cbe288a519f7bc5c406f4e..d4b079bef79254f03cc69bd6a8bc98c6b1fa20b7 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -202,6 +202,7 @@ component("modules") {
     "//third_party/webrtc_overrides:webrtc_component",
     "//third_party/zlib",
     "//tools/v8_context_snapshot:buildflags",
+    "//oc_fingerprint",
   ]
 
   public_deps = sub_modules
