diff --git a/services/network/network_context.cc b/services/network/network_context.cc
index 48d5d5c0eb741ea1bb9b1f25233574716a77b49a..45f17ec2a8752f5c4a184d1c78e2d5e677e71fab 100644
--- a/services/network/network_context.cc
+++ b/services/network/network_context.cc
@@ -4,6 +4,8 @@
 
 #include "services/network/network_context.h"
 
+#include "net/third_party/quiche/src/quiche/http2/core/spdy_protocol.h"
+
 #include <algorithm>
 #include <memory>
 #include <optional>
@@ -2969,6 +2971,13 @@ URLRequestContextOwner NetworkContext::MakeURLRequestContext(
 #endif  // BUILDFLAG(ENABLE_REPORTING)
 
   net::HttpNetworkSessionParams session_params;
+  // oc_fingerprint START
+  if (params_->http2_settings) {
+    for (const auto& [id, value] : *params_->http2_settings) {
+      session_params.http2_settings[static_cast<spdy::SpdySettingsId>(id)] = value;
+    }
+  }
+  // oc_fingerprint END
   bool is_quic_force_disabled = false;
   if (network_service_ && network_service_->quic_disabled()) {
     is_quic_force_disabled = true;
