diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 03fe6b86452b35c12fabdb7859fc0334c4af7caf..6d99520fffbaa5f62dd3b3b5964854217285429b 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1085,6 +1085,9 @@ static_library("ui") {
       "focus_tab_after_navigation_helper.h",
       "intent_picker_tab_helper.cc",
       "intent_picker_tab_helper.h",
+      "ocbot/ocbot_constants.h",
+      "ocbot/ocbot_side_panel_manager.cc",
+      "ocbot/ocbot_side_panel_manager.h",
       "read_anything/read_anything_prefs.cc",
       "read_anything/read_anything_prefs.h",
       "sad_tab.cc",
