diff --git a/extensions/common/mojom/api_permission_id.mojom b/extensions/common/mojom/api_permission_id.mojom
index e19de6942981b9667adbddde1ec8ae12d1262af4..5b62908a20a7a40c49de271ff5d500d1fe23cc87 100644
--- a/extensions/common/mojom/api_permission_id.mojom
+++ b/extensions/common/mojom/api_permission_id.mojom
@@ -288,6 +288,8 @@ enum APIPermissionID {
   kOmniboxDirectInput = 261,
   kExperimentalActor = 262,
   kEnterpriseLogin = 263,
+  kDeleted_Ocbot = 264,
+  kOcbot = 265,
 
   // Add new entries at the end of the enum and be sure to update the
   // "ExtensionPermission3" enum in
