diff --git a/chrome/browser/ui/startup/startup_tab_provider.cc b/chrome/browser/ui/startup/startup_tab_provider.cc
index 41a7a50939bf8f4bc1f33eb18258b4c49f91c112..3cc03b4e2acff61ab330501cd25839d17ee5bc91 100644
--- a/chrome/browser/ui/startup/startup_tab_provider.cc
+++ b/chrome/browser/ui/startup/startup_tab_provider.cc
@@ -375,7 +375,8 @@ StartupTabs StartupTabProviderImpl::GetNewTabPageTabsForState(
     const SessionStartupPref& pref) {
   StartupTabs tabs;
   if (!pref.ShouldRestoreLastSession()) {
-    tabs.emplace_back(GURL(chrome::kChromeUINewTabURL));
+    // ocbot: Use oc://home instead of chrome://newtab
+    tabs.emplace_back(GURL(chrome::kOcHomeURL));
   }
   return tabs;
 }
