diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7e7ea8254bd1b9d6f028e9d637713dad0f8ede25..4b2e154a1bfc97b8f238dbe59d482d94d921bb9f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2567,6 +2567,8 @@ static_library("browser") {
     "//ui/strings",
     "//ui/webui",
 
+    "//oc_fingerprint",
+
     # TODO(crbug.com/430804948): Remove this dependency when profiles/profile_manager.h
     # gets componentized.
     "//chrome/browser/ui/profiles:impl",
