diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 61ea8e60d7d26433a847c208aff58632e04d61df..348c5e0d15ee2bf4cd12e0e5a00b374988f03381 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2625,6 +2625,8 @@ static_library("browser") {
     "//ui/strings",
     "//ui/webui",
 
+    "//oc_fingerprint",
+
     # TODO(crbug.com/430804948): Remove this dependency when profiles/profile_manager.h
     # gets componentized.
     "//chrome/browser/ui/profiles:impl",
