// Add ocbot side panel action ID

--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -573,7 +573,8 @@
   E(kActionSidePanelShowShoppingInsights) \
   E(kActionSidePanelShowSideSearch) \
   E(kActionSidePanelShowUserNote) \
-  E(kActionSidePanelShowMerchantTrust)
+  E(kActionSidePanelShowMerchantTrust) \
+  E(kActionSidePanelShowOcbot, IDC_TOGGLE_OCBOT_SIDEPANEL)
 
 #define TOOLBAR_PINNABLE_ACTION_IDS \