diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index 0813930230f263520478e6fa0c7a3339e63cae52..ce71e174d20f0b745f51e8b4245d5ec150888f10 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -445,6 +445,7 @@ component("core") {
     "//ui/gfx/geometry",
     "//ui/gfx/geometry:geometry_skia",
     "//ui/strings",
+    "//oc_fingerprint",
   ]
 
   if (is_mac) {
